build/webpack.common.js

```diff
@@ -83,16 +82,12 @@ const withStylingModuleLoader = returnModuleRuleWithConfig(
 
 const withJSModuleLoader = returnModuleRuleWithConfig(
   {
-    test: /\.(jsx|js)?$/,
+    test: /\.(jsx|js|tsx|ts)?$/,
     exclude: /(node_modules)/,
   },
   [
     {
-      loader: 'babel-loader',
-      options: {
-        cacheDirectory: true,
-        presets: babelPresets,
-      },
+      loader: 'ts-loader',
```

Comment:
`ts-loader` will superceed babel, but how/where is the equivelent configuration for browser support (for example) configured?

Reply:
Perhaps! Curios how `"target": "es5",` maps to browsers

Reply:
It transpiles the code to es5 which is supported by all major browsers